htlcswitch/payment_result.go

```diff
@@ -39,6 +39,11 @@ type PaymentResult struct {
 	// irrevocably canceled. If the payment failed during forwarding, this
 	// error will be a *ForwardingError.
 	Error error
+
+	// EncryptedError will contain the raw bytes of an encrypted error
```

Comment:
does the switch currently have no access to the shared secret to decode it, but it was done by the channel router ?